### DISCUSSION TITLE
Why the "Streaming with Suspense"  in official App-Rourer demo deployed in vercel not work

### DISCUSSION THREAD

**Discussion by AndyBoat** [vercel/next.js#53739] [App Router] [answered]
When I visited the officail app-router demo of Streaming with Suspense , I ran into error report, both on Edge Runtime and Node Runtime. 

Detail:
site: https://app-router.vercel.app/streaming
screenshoot:
<img width="959" alt="image" src="https://github.com/vercel/next.js/assets/8624194/9e706d7a-6d1b-43e1-8acd-fdc536a864cc">
error msg : 
> Application error: a server-side exception has occurred (see the server logs for more information).
Digest: 2219929594

error msg in console:
> 6473-8c2b72212b579295.js:1 Error: An error occurred in the Server Components render. The specific message is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error.

**Comment by delbaoliveira** [comment, marked as answer]
Thank you for spotting this, it's now fixed: https://github.com/vercel/app-playground/pull/145
